setup/providers/kubernetes-v2/index.md

```diff
@@ -93,6 +93,13 @@ hal config provider kubernetes account add my-k8s-v2-account \
     --context $(kubectl config current-context)
 ```
 
+You'll also need to run the following
+
+```
+hal config features edit --artifacts true
+hal deploy apply
```

Comment:
Running `hal deploy apply --service-names deck` and `hal deploy apply --service-names clouddriver front50` didn't work. Redeploying everything did though.

Reply:
I would omit the `hal deploy apply` here, since the other docs assume this is part of a larger flow where `hal deploy apply` gets called at the end.

Reply:
Sounds good.